[BE-2915]: Added missing component documentation (#576)
appcircleio/appcircle-docusaurus: docs/publish-integrations/ios-publish-integrations/check-app-store-release-status.md

---
title: Check App Store Release Status
description: Learn how to get approval from TestFlight in Appcircle
tags: [application, app store, app store connect, app store version, app store status]
sidebar_position: 5
---

import Screenshot from '@site/src/components/Screenshot';

# Check App Store Release Status

> With Appcircle's **Check App Store Release Status** component, you can get the status information of your published application, bind this step to a condition according to the status, and check and run your flow.

Reviewer: ```suggestion
With Appcircle's **Check App Store Release Status** component, you can obtain status information for your published application, bind this step to a condition based on the status, and check and run your flow accordingly.
```

### Prerequisites

> Below are the prerequisite steps necessary for this operation, accompanied by their descriptions.

Reviewer: ```suggestion
Below are the prerequisite steps necessary for this operation, along with their descriptions.
```

| [**Sent to Testflight**](/publish-integrations/ios-publish-integrations/send-to-app-store)                           | This step allows you to submit your application to TestFlight.                                                                            |
| [**Get Approval from TestFlight**](/publish-integrations/ios-publish-integrations/approval-test-flight)              | This step checks the TestFlight status of your application and advances the Publish Flow according to the specified acceptance condition. |
| [**Add for Review on App Store Connect**](/publish-integrations/ios-publish-integrations/add-for-review-on-app-store) | This step checks the Release status of your application and advances the Publish Flow according to the specified acceptance condition.    |

<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2915-checkReleaseStatus.png' />

### Input Variables

Below are the parameters necessary for this step's operation, along with their descriptions.

<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2915-checkReleaseDetails.png' />

:::caution Success Statuses

> For this step to work depending on a condition, a valid status must be given. You can give more than one status by putting commas between them. Some statuses that can be given are listed below. For more information, please see the Apple documentation [**here**](https://developer.apple.com/help/app-store-connect/reference/app-and-submission-statuses/).

Reviewer: ```suggestion
For this step to work based on a condition, a valid status must be provided. You can specify multiple statuses by separating them with commas. Some example statuses are listed below. For more information, please refer to the [**Apple documentation**](https://developer.apple.com/help/app-store-connect/reference/app-and-submission-statuses/).
```
